site/content/uninstall.md

```diff
@@ -81,63 +81,63 @@ Complete the following steps on each host where you've installed the NGINX Agent
 
 ### Uninstalling NGINX Agent on SLES
 
-Complete the following steps on each host where you've installed the NGINX Agent:
+Complete the following steps on each host where you've installed NGINX Agent:
 
-1. Stop the NGINX Agent:
+1. Stop NGINX agent:
```

Comment:
Hey, is the  `a` supposed to be lowercase for NGINX Agent ?

Reply:
So:

If it's referring to NGINX Agent (the product): `Stop NGINX Agent`
If it's referring to an agent process for NGINX: `Stop the NGINX agent`

Reply:
Aww right makes sense, Thank you